package.json

```diff
@@ -72,6 +72,7 @@
     "cheerio": "^0.19.0",
     "coveralls": "2.11.2",
     "dateformat": "^1.0.8",
+    "ejs": "^2.3.3",
```

Comment:
Handlebars is very restrictive. I would have had to create a bunch of helpers, and I just didn't want to to go through that. EJS is only a dev dependency, so no harm.